packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -395,13 +395,25 @@ class RetainingPathWidget extends StatelessWidget {
   /// parentMapKey, and parentField where applicable.
   String _retainingObjectDescription(RetainingObject object) {
     if (object.parentListIndex != null) {
-      final ref = object.value as InstanceRef;
-      return 'Retained by element [${object.parentListIndex}] of ${ref.classRef?.name ?? '<parentListName>'}';
+      final ref = object.value;
+      if (ref is InstanceRef) {
+        return 'Retained by element [${object.parentListIndex}] of'
+            ' ${ref.classRef?.name ?? '<parentListName>'}';
+      } else {
+        return 'Retained by element [${object.parentListIndex}] of '
+            '${_objectName(ref) ?? '<unknown>'}';
+      }
     }
 
     if (object.parentMapKey != null) {
-      final ref = object.value as InstanceRef;
-      return 'Retained by element at [${_objectName(object.parentMapKey)}] of ${ref.classRef?.name ?? '<parentMapName>'}';
+      final ref = object.value;
+      if (ref is InstanceRef) {
+        return 'Retained by element at [${_objectName(object.parentMapKey)}] of'
+            ' ${ref.classRef?.name ?? '<parentMapName>'}';
+      } else {
+        return 'Retained by element at [${_objectName(object.parentMapKey)}] of '
```

Comment:
What case results in us following this path (here and elsewhere)?

Reply:
I found there was a case where the RetainingObject 'value' was a ContextRef, so it was throwing an error since that case was not being included